docs-web/configuration/security.md

```diff
@@ -0,0 +1,42 @@
+# Security
+
+We consider the security of the Ingress Controller paramount to the success of our users and use the following to 
+ensure the security of the Ingress Controller:
+* [Secure Development Life Cycle](https://www.microsoft.com/en-us/securityengineering/sdl/)
+* [FOSSA](https://fossa.com) scanning
+
+However, the Ingress Controller is deployed by a User in their environment, and as such, the User takes responsibility 
+for securing a *deployment* of the Ingress Controller. 
+We highly recommend every User to read and understand the following security concerns.
+
+## Kubernetes
+We recommend the Kubernetes [guide to securing a cluster](https://kubernetes.io/docs/tasks/administer-cluster/securing-a-cluster/).
+In addition, the following relating more specifically to Ingress Controller.
+
+### RBAC and Service Account
+The Ingress Controller is deployed within a Kubernetes environment, this environment must be secured. 
+Kubernetes uses [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) to control which types of users 
+can access which resources and what operations they can perform. 
+The Ingress Controller requires a service account which is configured using RBAC. 
+We strongly recommend using the [RBAC configuration](https://github.com/nginxinc/kubernetes-ingress/blob/master/deployments/rbac/rbac.yaml) provided in our standard deployment configuration. 
+It is configured with the least amount of privilege required for the Ingress Controller to work.
+
+### Certificates and Privacy Keys
+Secrets are required by the Ingress Controller for some configurations. 
+[Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) are stored by Kubernetes unencrypted by default. 
+We highly recommend configuring Kubernetes to store these Secrets encrypted at rest. 
```

Comment:
I'd nearly be tempted to drop all the 'highly' and 'strongly' in front of recommend. We're the people that wrote the software - if we're recommending you do something to make the software secure, you better treat that recommendation as a strong one.

Reply:
updated